### PR TITLE
Add Kinyarwanda and language selector to keycloak theme

### PR DESCRIPTION
**Pull request by jniweteto**

### CODE REVIEW

**Review by joshjabour — APPROVED** [review]
Looks good, and works well.

Noting the last few steps for this ticket:
1. Attempt to fix the configuration page (always reverts languages shown when rw is added)
2. Load to QA and test with Front end template
3. Demo for Sixbert and get feedback

File: themes/oaf/login/login.ftl [hunk 1]
```diff
@@ -109,6 +109,21 @@
                                     </div>
                                 </#if>
                             </form>
+                            <#if realm.internationalizationEnabled  && locale.supported?size gt 1> 
+                                <div id="kc-locale">
+                                    <div id="kc-locale-wrapper" class="${properties.kcLocaleWrapperClass!} whitney-font-400"">
```
Comment (joshjabour) [inline themes/oaf/login/login.ftl]: Delete extra double quote
```suggestion
                                    <div id="kc-locale-wrapper" class="${properties.kcLocaleWrapperClass!} whitney-font-400">
```

**Review by joshjabour — CHANGES_REQUESTED** [review]
1 minor change requested. Otherwise looks and works great!

**Review by joshjabour — APPROVED** [review]
Looks good to me!

**Review by bisonlou — APPROVED** [review]

**Pull request merged**